p2p/src/utils/mux.rs

```diff
@@ -131,7 +156,15 @@ impl<E: Spawner, S: Sender, R: Receiver> Muxer<E, S, R> {
 
                     // Get the route for the subchannel.
                     let Some(sender) = self.routes.get_mut(&subchannel) else {
-                        // Drops the message if the subchannel is not found
+                        // Attempt to use the backup channel if available.
+                        if let Some(backup) = &mut self.backup {
+                            if let Err(e) = backup.send((subchannel, (pk, bytes))).await {
+                                debug!(?subchannel, ?e, "failed to send message to backup channel");
+                            }
+                        }
+
+                        // Drops the message if the subchannel is not found or the backup
+                        // channel was used.
                         continue;
                     };
 
```

Comment:
If we want this to be "comprehensive", we should also send in the failure case just below?